themes/default/content/docs/guides/pulumi-packages/how-to-author.md

```diff
@@ -116,3 +131,9 @@ From there, a Pulumi employee will work with you to get your Pulumi Package publ
 ## Congratulations
 
 Congratulations on publishing your Pulumi Package!
+
+[^1]: Pulumi packages consist of SDKs, as well as a binary to facilitate the actual task (creating cloud resources, ect). Package
```

Comment:
```suggestion
[^1]: Pulumi packages consist of SDKs, as well as a binary to facilitate the actual task (creating cloud resources, etc.). Package
```

One thing on this, though: this seems like important information that package authors pretty much _need_ to know, so I probably wouldn't use a footnote for it. It probably deserves its own subheading, but for now, you could probably just tack it onto the end of this section to reduce the chances of its getting missed. What do you think?